examples/.ruff.toml

```diff
@@ -1,9 +1,11 @@
 extend = "../pyproject.toml"
+exclude = ["notebooks/*.ipynb"]
```

Comment:
ruff and jupytext sometimes disagree on formatting. 
but our ipynbs are autogeneratred anyways.